src/tests/ftest/scripts/setup_nodes.sh

```diff
@@ -71,6 +71,8 @@ rm -f /tmp/test.cov
 if [ -f /usr/lib/daos/TESTING/ftest/test.cov ]; then
     cp /usr/lib/daos/TESTING/ftest/test.cov /tmp
     chmod 777 /tmp/test.cov
+##DH++
+    export COVFILE="/tmp/test.cov"
```

Comment:
I'm not entirely sure what the goal is here as I know zilch about how Bullseye operates, but in any case, this is likely not doing what you are wanting as the life of this variable ends at the end of this script.  No further processing in the running of functional testing will see this variable.

I suspect what you really are looking for is to set this variable between these two lines: https://github.com/daos-stack/daos/blob/f51c272ce6cd4afea05ae4b39cc1694dc34be7ed/ftest.sh#L121-L122

Reply:
Do we need to export `COVFILE` here?  Launch.py sets this env at https://github.com/daos-stack/daos/blob/master/src/tests/ftest/launch.py#L1156

Reply:
It would seem not if `launch.py` is setting it itself.  I didn't know it was doing that.